<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Element found]]>
    </line>
    <line>
      <![CDATA[title not displayed]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="24143" started-at="2020-04-18T16:13:38 IST" finished-at="2020-04-18T16:14:02 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="24143" started-at="2020-04-18T16:13:38 IST" finished-at="2020-04-18T16:14:02 IST">
      <class name="regressionTestScripts.InvalidLogin">
        <test-method status="PASS" signature="setUp()[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a]" name="setUp" is-config="true" duration-ms="304" started-at="2020-04-18T16:13:37 IST" finished-at="2020-04-18T16:13:38 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="openAppn(java.lang.String)[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a]" name="openAppn" is-config="true" duration-ms="9531" started-at="2020-04-18T16:13:38 IST" finished-at="2020-04-18T16:13:47 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAppn -->
        <test-method status="FAIL" signature="invalidLogin()[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a]" name="invalidLogin" duration-ms="13139" started-at="2020-04-18T16:13:47 IST" finished-at="2020-04-18T16:14:00 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at generic.Base_Page.verifyTitle(Base_Page.java:33)
at regressionTestScripts.InvalidLogin.invalidLogin(InvalidLogin.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.TestRunner$$Lambda$56/1296674576.accept(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1234)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Element found]]>
            </line>
            <line>
              <![CDATA[title not displayed]]>
            </line>
          </reporter-output>
        </test-method> <!-- invalidLogin -->
        <test-method status="PASS" signature="closeAppn(org.testng.ITestResult)[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a]" name="closeAppn" is-config="true" duration-ms="1402" started-at="2020-04-18T16:14:00 IST" finished-at="2020-04-18T16:14:02 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidLogin status=FAILURE method=InvalidLogin.invalidLogin()[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a] output=Element found]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeAppn -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:regressionTestScripts.InvalidLogin@51cdd8a]" name="tearDown" is-config="true" duration-ms="846" started-at="2020-04-18T16:14:02 IST" finished-at="2020-04-18T16:14:03 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- regressionTestScripts.InvalidLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
